src/univers/swift_version_range.py

```diff
@@ -0,0 +1,57 @@
+import attr
+
+from univers.version_constraint import VersionConstraint
+from univers.version_range import VersionRange
+from univers.versions import SwiftVersion
+
+
+@attr.s(auto_attribs=True, frozen=True)
+class SwiftVersionRange(VersionRange):
+    expression: str
+    scheme: str = "swift"
+    version_class: type = SwiftVersion

```

Comment:
How about using `SemverVersion`, if that's what the Swift version is?

```suggestion
    version_class: type = versions.SemverVersion

```